Firestore/core/test/firebase/firestore/immutable/sorted_map_test.cc

```diff
@@ -297,6 +298,9 @@ TYPED_TEST(SortedMapTest, MinMax) {
 }
 
 TYPED_TEST(SortedMapTest, IteratorsAreDefaultConstructible) {
+  ASSERT_TRUE(
```

Comment:
Can you use `static_assert`?

Reply:
`StaticAssertTypeEq` doesn't work I'm not asserting two types are equal, merely some property of the type.

Changed to `static_assert`